### DISCUSSION TITLE
GBC TV

### DISCUSSION THREAD

**Discussion by youngbluespring** [iptv-org/community#340] [Show and tell]
**Channel Name:** GBC TV
**Channel ID:** GBCTV.gi
**Country:** Gibraltar
**Language:** English
**Website:** https://www.gbc.gi/tv/watch-live
**Stream URL (optional):** https://playback2.akamaized.net/streams/14143407_9828641_lsiwbf2g7pxtw1xejfy_1/exp=1637321502~acl=%2f*~data=hdntl~hmac=cd773424d94619e69cff55cf4b0700bfdbe8f7ef001bce419142eec48f26b3a5/media/14143407_9828641_lsiwbf2g7pxtw1xejfy_1@5256000p.m3u8?dw=14400&ts=1637233200
**Notes (optional):** xxx

**Comment by fotvnetwork** [comment]
Tokenized links

**Comment by Sphinxroot** [comment]
this one is odss 

if i use this in streamlink on kodi it work  https://livestream.com/accounts/14143407/events/9828641/player?width=960&height=540&enableInfoAndActivity=true&defaultDrawer=feed&autoPlay=true&mute=false
![gbc](https://user-images.githubusercontent.com/47762050/142708185-e9011c0e-a3b1-4996-a1cf-8e8ac45516d0.PNG)



but 
https://livestream.com/accounts/14143407/events/9828641  = 404

https://api.new.livestream.com/accounts/14143407/events/9828641/live.m3u8  event with referer and or user agent  it dont work

**Comment by BellezaEmporium** [comment]
Easy : 

https://player-api.new.livestream.com/accounts/14143407/events/9828641/stream_info

-> get the link from there.

I always picked the stream from this kind of links, on iptv-all.
